wiki/Submission/en.md

```diff
@@ -20,7 +20,7 @@ Two checkboxes appear near the bottom of the window. The first says `Receive ema
 
 ## Limitations
 
-Beatmaps will fail to submit if they exceed the online file size limit. That limit is 5MB plus an additional 10MB for every minute of beatmap length, and it caps at 100MB.
+Beatmaps will fail to submit if they exceed the online file size limit. That limit is 10MB plus an additional 10MB for every minute of mp3 length, and caps at 100MB. The beatmap can also fail to submit if there is more than 128 difficulties.
```

Comment:
cap has nothing to do with mp3, and the 5mb base limit is correct
\+ some wording
```suggestion
Beatmaps will fail to submit if they exceed the online file size or difficulty limit. The file size limit is 5MB plus an additional 10MB for every minute of beatmap length, and it caps at 100MB. The difficulty limit is currently 128.
```

Reply:
I think a compromise of 7mb should be done
![image](https://user-images.githubusercontent.com/37494321/71426180-b8587480-26f1-11ea-97b3-13698863a894.png)

Reply:
current values are correct as discussed on discord

Reply:
is my suggestion for this okay?